tests/CLI/CLI.h

```diff
@@ -55,4 +55,13 @@ struct CompleteIncompleteStruct;
 typedef struct IncompleteStruct IncompleteStruct;
 
 DLL_API IncompleteStruct* createIncompleteStruct();
-DLL_API void useIncompleteStruct(IncompleteStruct* a);
\ No newline at end of file
+DLL_API void useIncompleteStruct(IncompleteStruct* a);
+
+#ifdef _WIN32
+void testMSStyleAssembly() {
+	__asm {
```

Comment:
We try keeping our tests as simple as possible. Please reduce the test to a single instruction and then use the line syntax (remove {} from the assembly block).